public/app/plugins/datasource/stackdriver/components/AliasBy.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { debounce } from 'lodash';
+
+export interface Props {
+  onChange: (alignmentPeriod) => void;
+  value: string;
+}
+
+export interface State {
+  value: string;
+}
+
+export class AliasBy extends Component<Props, State> {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
don't bind in constructor, use arrow syntax in function declaration instead. Also name event callbacks with "on" prefix. example: "onChange"

Reply:
👍